quinn/examples/server.rs

```diff
@@ -97,18 +96,11 @@ fn run(log: Logger, options: Opt) -> Result<()> {
         server_config.use_stateless_retry(true);
     }
 
-    let keys = {
-        let mut reader =
-            io::BufReader::new(fs::File::open(&options.key).context("failed to read private key")?);
-        pemfile::rsa_private_keys(&mut reader).map_err(|_| err_msg("failed to read private key"))?
-    };
-    let cert_chain = {
-        let mut reader = io::BufReader::new(
-            fs::File::open(&options.cert).context("failed to read private key")?,
-        );
-        pemfile::certs(&mut reader).map_err(|_| err_msg("failed to read certificates"))?
-    };
-    server_config.set_certificate(cert_chain, keys[0].clone())?;
+    let key = fs::read(&options.key).context("failed to read private key")?;
+    let key = quinn::PrivateKey::from_pem(&key)?;
+    let cert_chain = fs::read(&options.cert).context("failed to read private key")?;
```

Comment:
The error is present in the old version as well but this should be changed to "failed to read certificates" or something like that.